spec/features/home/index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe "Home page" do
+  it "should show shop name and image" do
```

Comment:
This is great! It makes sense to have a home page, and so you should have a test for it.